spec/ability/roles_spec.rb

```diff
@@ -44,6 +44,14 @@
     FactoryGirl.create(:flagged_scanned_resource, user: image_editor, identifier: ['ark:/99999/fk4445wg45'])
   }
 
+  let(:complete_ephemera_folder) {
```

Comment:
Yes, I meant to check to make sure anonymous users could `manifest` them. Added it.